SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -721,7 +721,10 @@ void USpatialReceiver::ReceiveActor(Worker_EntityId EntityId)
 			return;
 		}
 
-		PackageMap->ResolveEntityActor(EntityActor, EntityId);
+		if (!PackageMap->ResolveEntityActor(EntityActor, EntityId))
```

Comment:
We should cleanup the actor and the channel here

Reply:
Added
`			EntityActor->Destroy(true);
			Channel->Close(EChannelCloseReason::Destroyed);
`

Reply:
nit: Can we an additional log here for this explicit instance?